docs/dev/testnet/status.md

```diff
@@ -3,7 +3,7 @@
 ## Currently supported features
 
 - **Native support of ECDSA signatures.** Unlike the first version of zkSync and most of ZK rollups, no special operation is required to register the user's private key. Any account can be managed on L2 with the same private key that is used on L1.
-- **Solidity 0.8.x support.** No need for change or re-audit of the codebase.
+- **Solidity 0.8.x and Vyper support.** No need for change or re-audit of the codebase.
```

Comment:
They are unstable. IDK, I wasn't given the green light to confidently talk about supporting them, so I mentioned them once in the documentation, but generally I try to avoid it

Reply:
but we made public announcements about it already